src/queues/job-queue.js

```diff
@@ -41,8 +44,8 @@ const {addUserPostCommentProcess} = require('../processes/add-user-post-comment'
 const {deleteUserPostCommentProcess} = require('../processes/delete-user-post-comment-process');
 const BetterSocialCronQueue = require('../redis/BetterSocialCronQueue');
 
-const Sentry = require('@sentry/node');
-const SentryProfiling = require('@sentry/profiling-node');
+const serviceAccount = Buffer.from(process.env.SERVICE_ACCOUNT, 'base64').toString();
+initializeApp({credential: cert(JSON.parse(serviceAccount))});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The service account is being initialized directly from an environment variable without any error handling. If the `SERVICE_ACCOUNT` environment variable is not set or is invalid, this will throw an error. It would be better to add some error handling around this.
```diff
- const serviceAccount = Buffer.from(process.env.SERVICE_ACCOUNT, 'base64').toString();
- initializeApp({credential: cert(JSON.parse(serviceAccount))});
+ try {
+   const serviceAccount = Buffer.from(process.env.SERVICE_ACCOUNT, 'base64').toString();
+   initializeApp({credential: cert(JSON.parse(serviceAccount))});
+ } catch (error) {
+   console.error('Failed to initialize Firebase Admin with service account:', error);
+   process.exit(1);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->